sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/TestEnvironment.cs

```diff
@@ -32,7 +33,7 @@ public static class TestEnvironment
         /// The relative path to the JPG file which contains the receipt to be used for tests.
         /// </summary>
         /// <value>The relative path to the JPG file.</value>
-        public static string ReceiptPath => Path.Combine(AssetsFolderName, ReceiptFilename);
+        public static string ReceiptPath => Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), AssetsFolderName, ReceiptFilename);
```

Comment:
Not sure why this is happening. It could be a good idea to do the same to `RetrieveInvoicePath`.

Reply:
This is the fix to make it work on my machine 😊
I'll update RetrieveInvoicePath as well.